libexec/pyenv-init

```diff
@@ -260,52 +253,40 @@ function print_shell_function() {
   commands=(`pyenv-commands --sh`)
   case "$shell" in
   fish )
-    cat <<EOS
-function pyenv
+    echo "function pyenv
```

Comment:
Can probably use single quotes here and further to save ourselves from having to track what to escape.